pages/linux/efibootbgr.md

```diff
@@ -0,0 +1,23 @@
+# efibootmgr
+
+> Manipulate the UEFI Boot Manager (the Bootoptions).
+
+- List the current settings:
+
+`efibootmgr `(`-v` for Filepaths)
+
+- Create new Bootoption:
+
+`sudo efibootmgr -c -d {{esp_blockdevice}} -l {{\path\to\file.efi}} -L "{{Label}}"`
```

Comment:
I'm a bit confused - doesn't Linux use forward slashes `/` in paths - perhaps this will read better?
```suggestion
`sudo efibootmgr -c -d {{esp_blockdevice}} -l {{/path/to/file.efi}} -L "{{Label}}"`
```

Reply:
The ESP Blockdevice has a FAT (dos) Filesystem, and FAT Filesystems are ("officially") using \.
I have no clue if you could use / as well, because I have never done it. And `efibootmgr -v` also gives out file paths like  `Boot0001* GRUB	HD(1,GPT,[UUID],0x800,0x20000)/File(\EFI\GRUB\grubx64.efi)`.

Reply:
Ah, I see. So this isn't a regular file path - it's a special ESP / boot filepath. That makes more sense then :-)